Incompatible With React-Router V4

The changes from react-router V3 to V4 has changed the expected file structure, mergeRouteParams.js no longer exists. I have looked through their source and am unable to find anything that has the same functionality as this file. I downgraded to 3.0.0 and it works fine, worth making a small note in the docs to use react-router < 4.0.0
``` Error in ./~/react-router-relay/lib/utils/mergeRouteParams.js
Module not found: 'react-router/lib/getRouteParams' in 
/Users/00y/code/malik.img/node_modules/react-router-relay/lib/utils

 @ ./~/react-router-relay/lib/utils/mergeRouteParams.js 11:22-64

--- a/README.md
+++ b/README.md
@@ -1,6 +1,8 @@
 # react-router-relay [![Travis][build-badge]][build] [![npm][npm-badge]][npm]
 [Relay](http://facebook.github.io/relay/) integration for [React Router](https://github.com/reactjs/react-router).
 
+**This library does not support React Router v4, because React Router v4 does not provide the necessary integration points for efficient data fetching with Relay. Additionally, rendering performance will be better with React Router v2 than with React Router v3, as the link subscription logic in React Router v3 triggers unnecessary rerenders with Relay.**
+
 [![Discord][discord-badge]][discord]
 
 ## Usage

